web/Views/Shared/_Layout.cshtml

```diff
@@ -26,6 +26,14 @@
                         <li class="nav-item">
                             <a class="nav-link text-dark" asp-area="" asp-controller="Home" asp-action="Privacy">Privacy</a>
                         </li>
+                        <li class="nav-item">
+                            @{
+                                if (User.Identity?.IsAuthenticated == true)
+                                {
+                                    <a class="nav-link text-dark" asp-area="" asp-controller="Landlord" asp-action="RegisterGet">Profile</a>

```

Comment:
You can check `((BricksAndHeartsUser)User.Identity).LandlordId != null` to see if a user is registered as a landlord yet. That feels a bit nicer than relying on "RegisterGet" to redirect and means we can hide the button if they're not a landlord (or change it to "Register")